### DISCUSSION TITLE
Too much objects IgnoredChar and SourceLocations had been created

### DISCUSSION THREAD

**Discussion by kofeine** [graphql-java/graphql-java#2489] [Q&A]
Can anyone guess how could it happens?
graphql-java 16.2
via graphql-spring-boot 11.1.0
(mem usage)
![image](https://user-images.githubusercontent.com/88098184/127548285-443443a1-a9a7-4924-a616-889d81c2b1d3.png)
![image](https://user-images.githubusercontent.com/88098184/127549818-1980d57c-fa4d-474d-a217-b618c0de5143.png)
![image](https://user-images.githubusercontent.com/88098184/127549963-cc3a598b-cab7-4260-8b42-e95792833acf.png)
![image](https://user-images.githubusercontent.com/88098184/127550118-60941827-f998-4fd4-adde-8caeb78ec895.png)

**Comment by bbakerman** [comment]
we have good news on this front

we have made options in he SchemaGen that allows you to turn this off

https://github.com/graphql-java/graphql-java/pull/2438/files

https://github.com/graphql-java/graphql-java/pull/2439


The default will be to NOT capture IgnoredChars.

This is coming in 17.0 which will be released soon

**Reply by kofeine** [reply]
Great! But, there is any way to fix it myself right now is my project?

**Comment by bbakerman** [comment]
No - is been fixed on master - the new release will be out soon.  That's
the only way to fix it

**Reply by softshape** [reply]
How soon is that? We can't update to 17.0 until it's beta as we're using graphql-spring-boot and its author won't update it until v17 is out of beta.

**Reply by andimarek** [reply]
17.0 is released now.

**Reply by softshape** [reply]
Great news! Can't wait to test it in production.
